BeeSwift/AppDelegate.swift

```diff
@@ -149,4 +149,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     private func removeAllLocalNotifications() {
         UNUserNotificationCenter.current().removeAllPendingNotificationRequests()
     }
+    func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
+        if #available(iOS 12.0, *) {
+            if let intent = userActivity.interaction?.intent as? AddDataIntent {
+                guard let value = intent.value, let slug = intent.goal else { return false }
+                RequestManager.addDatapoint(urtext: "^ \(value)", slug: slug) { (response) in
```

Comment:
Where can documentation be found explaining this shorthand here?

setting
urText to "^ value" ?



(^ is translated on the server to mean "today" and value is the value being added as part of the datapoint, right? And one can write directly to urText rather than specifying daystamp and value individually, as outlined in the [api docs](https://api.beeminder.com/#postdata).)